presto-main/src/main/java/com/facebook/presto/operator/PagesHash.java

```diff
@@ -238,4 +241,18 @@ private static int getHashPosition(long rawHash, long mask)
 
         return (int) (rawHash & mask);
     }
+
+    /**
+     * @return Given a number of positions, how much more memory would a pagesHash use if we constructed it from a PagesIndex with this number of positions?
+     * Keep in mind that this is a *Delta*: The actual pagesHash would be larger than this -- add this number to the existing size of the PagesIndex
+     */
+    static long calculatedPredictedSizeDelta(int positionCount){
+        int hashSize = HashCommon.arraySize(positionCount, 0.75f);
```

Comment:
Please extract it into a separate method (e.g.: getHashTableSize), and use it in both places, so there's less chance that somebody will change the first one and wont change the other.

Reply:
(y)